configure.ac

```diff
@@ -19,7 +19,13 @@ m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
 # Checks for libraries.
 PKG_CHECK_MODULES(OPENSSL, [libssl >= 0.9.8 libcrypto >= 0.9.8], [], [AC_MSG_ERROR([Cannot find OpenSSL 0.9.8 or higher.])])
-AC_CHECK_LIB([pthread], [pthread_create], [], [AC_MSG_ERROR([Cannot find libpthread.])])
+AC_MSG_CHECKING([POSIX threads support])
+AX_PTHREAD([
+	CLIBS="$LIBS $PTHREAD_LIBS"
+	CFLAGS="$CFLAGS $PTHREAD_CFLAGS"
+	LDFLAGS="$LDFLAGS $PTHREAD_CFLAGS"
```

Comment:
should this perhaps be `LDFLAGS="$LDFLAGS $PTHREAD_LDFLAGS"` here?

Reply:
Mmmh... Probably!